src/Dialogs/OpenAttachmentDialog.vala

```diff
@@ -61,16 +69,14 @@ public class Mail.AttachmentDialog : Gtk.Dialog {
                 show_file_anyway.begin ();
             }
         });
-
-        show_all ();
     }
 
     private async void show_file_anyway () {
         try {
             GLib.FileIOStream iostream;
             var file = File.new_tmp ("XXXXXX-%s".printf (mime_part.get_filename ()), out iostream);
             yield mime_part.content.decode_to_output_stream (iostream.output_stream, GLib.Priority.DEFAULT, null);
-            yield GLib.AppInfo.launch_default_for_uri_async (file.get_uri (), null, null);
+            yield GLib.AppInfo.launch_default_for_uri_async (file.get_uri (), new AppLaunchContext (), null);
```

Comment:
The binding here is misleading, it's nullable so If you want to avoid the compilation warning, better do `(GLib.AppLaunchContext) null`

Reference: https://developer.gnome.org/gio/stable/GAppInfo.html#g-app-info-launch-default-for-uri-async